docs/guides/lido-tokens-integration-guide.md

```diff
@@ -8,9 +8,14 @@ Lido is a family of liquid staking protocols across multiple blockchains, with h
 Liquid refers to the ability of a user’s stake to become liquid. Upon the user's deposit Lido issues stToken, which represents the deposited tokens along with all the rewards & penalties accrued through the deposit's staking. Unlike the staked funds, this stToken is liquid — it can be freely transferred between parties, making it usable across different DeFi applications while still receiving daily staked rewards. It is paramount to preserve this property when integrating stTokens into any DeFi protocol.
 
 This guide refers to Lido on Ethereum (hereinafter referred to as Lido).
-For ether staked in Lido, it gives users stETH that is equal to the amount staked.
 
-Lido's stTokens are widely adopted across the Ethereum ecosystem:
+## Lido tokens
+
+### stTokens: stETH and wstETH
+
+For ether staked in Lido, it gives users [stETH](#steth) that is equal to the amount staked. Since the `stETH` token is rebasable, there is also its value-accruing wrapped version [wstETH](#wsteth) to ease DeFi integrations.
```

Comment:
> For ether staked in Lido, it gives users [stETH](#steth) that is equal to the amount staked.

Ambiguous "it". Usually, it's much better to specify the subject explicitly, e.g., "Staking ether gives..."

> Since the `stETH` token is rebasable, there is also its value-accruing wrapped version [wstETH](#wsteth) to ease DeFi integrations.

"There is ..." is a bit too dry and boring. Opt to use dynamic verbs, e.g.,
"For easier DeFi integrations, stETH has a non-rebaseable counterpart called wstETH."